src/hydro/hydro.cpp

```diff
@@ -135,6 +135,7 @@ void PreStepMeshUserWorkInLoop(Mesh *pmesh, ParameterInput *pin, SimTime &tm) {
     // Finally update c_h
     const auto &cfl_hyp = hydro_pkg->Param<Real>("cfl");
     const auto &dt_hyp = hydro_pkg->Param<Real>("dt_hyp");
+    mindx = hydro_pkg->Param<Real>("mindx");
```

Comment:
Maybe this variable could be renamed? That would have avoided the original bug.

Reply:
This logic should eventually be moved to function that's only triggered on mesh refinement (as only then mindx changes).
I'll update it then if that's okay.